packages/jest-expect-har/.eslintrc

```diff
@@ -1,13 +1,4 @@
 {
-  "extends": [
-    "@readme/eslint-config",
-    "@readme/eslint-config/typescript"
-  ],
+  "extends": ["@readme/eslint-config", "@readme/eslint-config/typescript"],
   "root": true,
```

Comment:
Retaining `root: true` in these two packages because the root loads in our ESM config and neither of these packages can be ESM because of Jest.